docs/html/cli/index.md

```diff
@@ -31,13 +31,6 @@ pip_wheel
 pip_hash
 ```
 
-```{toctree}
-:maxdepth: 1
-:caption: Package Index information
-
-pip_search
```

Comment:
I wonder if this would trigger the orphan page warning. (Not sure how to avoid it if so.)

Reply:
Yes, it triggers an orphan warning. It can be silenced by adding `:orphan:` at the top of the page.

It's only referenced in `main` at: 
* https://github.com/pypa/pip/blob/b1a8539c6a6cf915ba10890c3dee847861644f71/docs/html/cli/index.md
* https://pip.pypa.io/en/stable/cli/

And the redirect at:

* https://github.com/pypa/pip/blob/b1a8539c6a6cf915ba10890c3dee847861644f71/docs/html/reference/pip_search.rst
* https://pip.pypa.io/en/stable/reference/pip_search/


Should `:orhpan:` be added? Or the pages removed completely?

---

Relatedly, looks like a `tox -e docs` job is missing from the CI, there's another warning that has slipped through:

```
Warning, treated as error:
/Users/hugo/github/pip/docs/html/user_guide.rst:1011:undefined label: dependency resolution backtracking
```

Reply:
@pradyunsg Could we add a redirect to `pip index`

I feel we can remove the page altogether and put a banner on top of the `pip index` page to tell users `pip search` has been removed and this is its replacement.

Reply:
From my point of view, adding `:orphan:` should be ok for the `pip search` docs failure. Not sure if we should remove the page by now, I would personally prefer a removal a few releases after. A redirection to `pip index` sounds a good idea to me.

Anyone else has thoughts about this?

Reply:
Any update on this?